.mailmap

```diff
@@ -1547,6 +1549,7 @@ dandiez <47832466+dandiez@users.noreply.github.com>
 der-blaue-elefant <github@kklein.de>
 dimasvq <dimas.vq.2020@bristol.ac.uk>
 dispasha <dispasha@users.noreply.github.com>
+dodo <palumbododo@gmail.com>
```

Comment:
@smichr please don't merge pull requests where the .mailmap file is being filled out incorrectly:
https://github.com/sympy/sympy/blob/8c94201570737a2fc3ef8e9cc53bed01a44e8281/.mailmap#L138-L148